### PR TITLE
Add support for OL8

### PR DESCRIPTION
**Pull request by jzuellig**
Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
The server role does not explicitly support OL8.

Issue Number: `e.g. #1` 

## What is the new behavior?
The server role explicitly supports OL8

### CODE REVIEW

**Review by robin-checkmk — CHANGES_REQUESTED** [review]

File: roles/server/tasks/RedHat.yml [hunk 1]
```diff
@@ -48,7 +48,16 @@
...
+    - enable-repos
+
+- name: "Enable powertools repository on OracleLinux 8."  # noqa no-changed-when
+  become: true
+  ansible.builtin.shell: dnf config-manager --set-enabled ol8_codeready_builder  # noqa command-instead-of-shell
+  when: | # RHEL needs epel and a package URL
+    ansible_facts.distribution == "OracleLinux"
     and ansible_distribution_major_version == "8"
   tags:
     - enable-repos
```
[the remark this answers is not in the document]
Comment (jzuellig) [reply roles/server/tasks/RedHat.yml]: Has now a more selfexplaing name 👍

**Comment by jzuellig** [comment]
> > If seLinux is enabled on my OL8 System, I get the following error:
> > ```
> > TASK [checkmk.general.server : Enable httpd can network connect selinux boolean.] **********************************************************************************************************************************
> > An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'selinux'
> > fatal: [s2ol8.local]: FAILED! => {"changed": false, "msg": "Failed to import the required Python library (libselinux-python) on localhost.localdomain's Python /usr/bin/python3.9. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > The task is based on ansible.posix.seboolean. Is finding a solution in the Scope of this project?
> 
> When the task was initially introduced, the installation of `python3-libsemanage` tackled that, but maybe the package has a different name now?

I'll investigate it tomorrow with the help of our Linux team

**Comment by jzuellig** [comment]
> I'll investigate it tomorrow with the help of our Linux team


We have found the cause: because I have installed python3.9 on my machine to run Ansible.
Ansible uses it, instead of the "platfrom-python"  not all needed packages are available for 3.9 under OL8.
I have resolved it by adding ansible_python_interpreter: ```/usr/libexec/platform-python3.6``` to my host in the inventory.

Therefor, I think this phenomenon is out of scope for this Ansible collection.



To the Failed checks… 
I'm fairly new to them, if I interpret the errors correctly there was a space issue. 
This is not an issue of my code / commits, right? 
Can I re-run them, or do you have to trigger them?

**Comment by robin-checkmk** [comment]
Thanks for the analysis!

Sounds like a corner case to me as well, so I think we are good to merge. If this comes up again, we can tackle the issue then.

Regarding the tests, there seems to be a problem with GitHub Actions. I will re-trigger them tomorrow and see, if it sorts itself out.

Thanks for your contribution, this will make it into the `2.4.0` release most certainly.
